packages/headless/src/components/popover/popover.tsx

```diff
@@ -52,7 +52,10 @@ interface PopoverProps {
 }
 
 export const Popover = component$((props: PopoverProps) => {
-  const { triggerEvent = 'click', onUpdate$, disableClickOutSide } = props;
+  const triggerEvent = props.triggerEvent ?? 'click';
+  const onUpdate$ = props.onUpdate$;
+  const disableClickOutSide = props.disableClickOutSide;
```

Comment:
original expression stopped working after migration to latest qwik

Reply:
what do you mean by "original expression"?

Reply:
```js
const { triggerEvent = 'click', onUpdate$, disableClickOutSide } = props;
```

there's a new error `props` has already been defined. has something to do with optimizer, I believe, will try to replicate and create a ticket in qwik later, if it's reproducible

Reply:
OK, nice catch!
yeah an issue is a great idea
thanks Dima!